src/main/java/net/dv8tion/jda/core/audio/AudioConnection.java

```diff
@@ -514,7 +515,7 @@ else if (sample < Short.MIN_VALUE)
                 }
                 catch (Exception e)
                 {
-                    LOG.fatal(e);
+                    LOG.error("Something happened!", e);
```

Comment:
Can we make the comments in this file a bit more descriptive like "Unexpected Exception occurred while X"

Reply:
:) Yea i didn't know what exactly happened there

Reply:
Its funny how you didn't mention the other 2 exceptions above :)

Reply:
I used the plural "comments" for a reason 👀